gramps/gen/config.py

```diff
@@ -23,7 +23,7 @@
 #
 
 """
-This package implements access to GRAMPS configuration.
+This package implements access to Gramps configuration.
 """
 
```

Comment:
Ok removed the editorial changes and pushed back to github and it closed !
https://github.com/sam-m888/gramps/commit/28f3d90f